.github/workflows/cypress-altinn-app-frontend.yml

```diff
@@ -55,13 +55,10 @@ jobs:
           export CYPRESS_PROJECT_ID=y2jhp6
           export CYPRESS_RECORD_KEY=${{ secrets.CYPRESS_RECORD_KEY }}
           export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start' http://localhost:8080/altinn-app-frontend.js 'yarn run test:appfrontend:headless --env environment=tt02,testUserName=tt02testuser,testUserPwd=${{ secrets.CYPRESS_ALTINN_USERPWD }} --record --parallel --tag "altinn-app-frontend" --group altinn-app-frontend --ci-build-id $GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT'
+          ./node_modules/.bin/start-server-and-test 'cd $GITHUB_WORKSPACE/src/altinn-app-frontend && yarn start' http://localhost:8080/altinn-app-frontend.js 'yarn run test:all:headless --env environment=tt02,testUserName=tt02testuser,testUserPwd=${{ secrets.CYPRESS_ALTINN_USERPWD }} --record --parallel --tag "altinn-app-frontend" --group altinn-app-frontend --ci-build-id $GITHUB_RUN_ID-$GITHUB_RUN_ATTEMPT'
         working-directory: test/cypress
 
   altinn-app-frontend-test-on-fork-pr:
-    if: |
```

Comment:
removing this has run the test twice. refer [this](https://github.com/Altinn/app-frontend-react/runs/6271678922?check_suite_focus=true).

Reply:
Yikes! I didn't even mean to remove that! Thanks @jeevananthank! I tried using using `act` to test the workflow locally and I had to remove this check to get it to work. Good thing we do code reviews!